src/otx/recipe/classification/multi_class_cls/deit_tiny.yaml

```diff
@@ -4,11 +4,6 @@ model:
     label_info: 1000
     arch: "deit-tiny"
 
-    loss_callable:
-      class_path: torch.nn.CrossEntropyLoss
-      init_args:
-        reduction: none
```

Comment:
@harimkang

Reply:
Thanks to share. I'll take a look at it later to see if there are any potential issues.

Reply:
Please create a JIRA ticket to clean up those things.